package/PartSeg/plugins/napari_widgets/search_label_widget.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, napari_viewer: Viewer):
         self.napari_viewer = napari_viewer
         self.search_type = create_widget(annotation=SearchType, label="Search type")
         self.search_type.changed.connect(self._component_num_changed)
-        self.component_selector = SpinBox(name="Label number", min=0)
+        self.component_selector = SpinBox(name="Label number", min=0, max=2**64 - 1)
```

Comment:
```suggestion
        self.component_selector = SpinBox(name="Label number", min=0, max=2**31 - 1)
```

Qt int has 4 bits https://bugreports.qt.io/browse/QTBUG-63765 and is unsigned int, so There is a need to lower the max value.